app/services/usps_in_person_proofing/mock/proofer.rb

```diff
@@ -29,7 +29,7 @@ def request_enroll(applicant)
         Response::RequestEnrollResponse.new(res)
       end
 
-      def request_facilities(_location)
+      def request_facilities(_location, _vector_of_trust)
```

Comment:
see previous comment. If the "real" proofer uses keyword args, this must as well, we'll likely have to tell Rubocop to ignore unused arguments
```suggestion
      def request_facilities(location: nil, vector_of_trust: nil) # rubocop:disable Lint/UnusedMethodArgument
```